<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="62067" started-at="2019-05-20T06:21:16Z" finished-at="2019-05-20T06:22:18Z">
    <groups>
    </groups>
    <test name="Test1" duration-ms="62067" started-at="2019-05-20T06:21:16Z" finished-at="2019-05-20T06:22:18Z">
      <class name="userTest.VerifyLogin">
        <test-method status="PASS" signature="reportSetUp()[pri:0, instance:userTest.VerifyLogin@36f6e879]" name="reportSetUp" is-config="true" duration-ms="189" started-at="2019-05-20T06:21:15Z" finished-at="2019-05-20T06:21:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportSetUp -->
        <test-method status="PASS" signature="initialization()[pri:0, instance:userTest.VerifyLogin@36f6e879]" name="initialization" is-config="true" duration-ms="53616" started-at="2019-05-20T06:21:16Z" finished-at="2019-05-20T06:22:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="PASS" signature="beforePrint()[pri:0, instance:userTest.VerifyLogin@36f6e879]" name="beforePrint" is-config="true" duration-ms="1" started-at="2019-05-20T06:22:09Z" finished-at="2019-05-20T06:22:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforePrint -->
        <test-method status="FAIL" signature="verifyLogin()[pri:0, instance:userTest.VerifyLogin@36f6e879]" name="verifyLogin" duration-ms="7497" started-at="2019-05-20T06:22:09Z" finished-at="2019-05-20T06:22:17Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Text not matched - Test failed expected [Quick Link] but found [Quick Links]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Text not matched - Test failed expected [Quick Link] but found [Quick Links]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at utility.CommonMethods.getText(CommonMethods.java:27)
at userTest.VerifyLogin.verifyLogin(VerifyLogin.java:15)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:userTest.VerifyLogin@36f6e879]" name="afterMethod" is-config="true" duration-ms="735" started-at="2019-05-20T06:22:17Z" finished-at="2019-05-20T06:22:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLogin status=FAILURE method=VerifyLogin.verifyLogin()[pri:0, instance:userTest.VerifyLogin@36f6e879] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:userTest.VerifyLogin@36f6e879]" name="tearDown" is-config="true" duration-ms="51" started-at="2019-05-20T06:22:18Z" finished-at="2019-05-20T06:22:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="extentClose()[pri:0, instance:userTest.VerifyLogin@36f6e879]" name="extentClose" is-config="true" duration-ms="1004" started-at="2019-05-20T06:22:18Z" finished-at="2019-05-20T06:22:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extentClose -->
      </class> <!-- userTest.VerifyLogin -->
    </test> <!-- Test1 -->
  </suite> <!-- Suite -->
</testng-results>
